packages/flutter/lib/src/painting/strut_style.dart

```diff
@@ -0,0 +1,500 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+import 'basic_types.dart';
+
+/// [StrutStyle] defines the strut, which defines the minimum height a line can be
```

Comment:
No need to repeat the name of the class here, it would make the docs look like this:

`StrutStyle: StrutStyle defines...`